docs/protocol/register_packages_on-chain.md

```diff
@@ -0,0 +1,88 @@
+This guide describes how to register software packages in the Autonolas Protocol using the [protocol frontend](https://protocol.autonolas.network/). You can register three types of packages: **services**, **agents** and **components**.
+
+## How packages are registered
+
+To register a package in the protocol, it must have been already published into a remote (IPFS) registry. Read [how to publish packages](https://docs.autonolas.network/open-autonomy/guides/publish_fetch_packages/) with the {{open_autonomy}} framework.
+
+The process of registering a package comprises the creation of a **metadata file** which contains a pointer to the IPFS hash of the published software package itself, plus some other additional information required by the protocol. This metadata file is also stored in the IPFS registry. The [protocol frontend](https://protocol.autonolas.network/) will automatically create and publish the metadata file once you provide the necessary information, and it will interact with the protocol smart contracts to complete the registration of your package.
+
+Upon completing the registration process, you will have minted on-chain an NFT representing your package. The process is summarized in the figure below.
+
+<figure markdown>
+![Package registration](./images/register_package.svg)
+</figure>
+
+## Requirements
+
+In order to register a software package, you must ensure that you have:
+
+* A **crypto wallet** (e.g., [Metamask](https://metamask.io/) or a cold wallet) with funds for the chain that you wish to register the package.
+* The **hash of the package** that you want to register on-chain, and which should be already published into a remote registry.
+* An **NFT image URL**. This image will be used to represent the minted NFT for the package on marketplaces such as [OpenSea](https://opensea.io/). You can use [this sample image URL](https://gateway.autonolas.tech/ipfs/Qmbh9SQLbNRawh9Km3PMEDSxo77k1wib8fYZUdZkhPBiev) for testing purposes.
+
+## Register a component
+
+Connect your wallet to the frontend, open the [components section](https://protocol.autonolas.network/components) and press the button _Register_. Fill in the data of your component:
+
+  1. **Owner Address.** The wallet address of the component owner (starting by `0x...`). It need not be the address of the wallet connected.
```

Comment:
I believe it's "starting at '0x...'". Also "it need not" --> "it does not need", "wallet connected" --> "connected wallet"

Reply:
I think "starting by" is ok.
I have changed the other 2 points globally.